README.md

```diff
@@ -81,6 +81,7 @@ Twitter              | Fediverse                           | Name
 @maxandersen         | @maxandersen@mastodon.social        | Max Rydahl Andersen
 @McPringle           | @McPringle@mastodon.social          | Marcus Fihlon
 @MGrzejszczak        | @marcin_grzejszczak@mastodon.social | Marcin Grzejszczak
+@mesirii             | @mesirii@chaos.social               | Michael Hunger
```

Comment:
The table is currently sorted by Twitter handles. May I ask you to move you one row up? Many thx!